### DISCUSSION TITLE
[Idea] Sepolia faucet support for OP

### DISCUSSION THREAD

**Discussion by shawnonchain** [ethereum-optimism/developers#31] [Ideas]
### Idea

Hey all! Shawn from Automata Network here. [Automata Network](https://ata.network/) is a modular attestation layer that extends machine-level trust to Web3 by leveraging optimistic rollups and zero-knowledge proofs in its Proof of Machinehood approach.

We are exploring to support OP on Sepolia faucet which will be backed by [Proof of Machinehood](https://docs.ata.network/automata-2.0/proof-of-machinehood) - that means users can get testnet tokens without the need of account sign ups, social verification or tasklist. We also got the direct support from Ethereum Foundation DevOps team to fund Sepolia and Holesky ETH. 

Feel free to try out the 2 testnet faucets on ETH here:
https://www.sepoliafaucet.io/
https://www.holeskyfaucet.io/

We would like to engage OP for some amplification support. What is the best way to move forward?

**Comment by sbvegan** [comment]
Hey @Shawn-ata! You can check out the [redirects on the readme](https://github.com/ethereum-optimism/developers#redirects-%EF%B8%8F), there's a couple forms there that you should be able to utilize. I'd start with the, "Connect with OP Labs" form because I believe the marketing form is for projects who have already launched on OP Mainnet. 

OP Mainnet has attestation support through [EAS](https://optimism.easscan.org/), would you all be able to hook into that for your faucet?
